lib/broadside/target.rb

```diff
@@ -74,14 +85,29 @@ def family
       "#{Broadside.config.application}_#{@name}"
     end
 
-    def to_h
+    def to_hash
```

Comment:
huh well i've definitely been using `to_hash` to investigate stuff, but if that's the case i'll just switch this.